ws-nextjs-app/pages/[service]/new_live/[id]/Post/styles.tsx

```diff
@@ -0,0 +1,18 @@
+import { Theme, css } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+export default {
+  postHeaderBanner: ({ palette }: Theme) =>
+    css({
+      'align-items': 'flex-start',
```

Comment:
This could be changed to `alignItems`

Reply:
I agree, I get a terminal error - `Using kebab-case for css properties in objects is not supported. Did you mean alignItems?`